analyzer/src/main/kotlin/managers/Bower.kt

```diff
@@ -96,11 +96,35 @@ class Bower(
                 }
             }
 
+        /**
+         * Parse information about the author. According to https://github.com/bower/spec/blob/master/json.md#authors,
+         * there are two formats to specify the authors of a package (similar to NPM). The difference is that the
+         * strings or objects are inside an array.
+         */
+        private fun extractAuthors(node: JsonNode): SortedSet<String> =
+            sortedSetOf<String>().apply {
+                node["pkgMeta"]["authors"]?.mapNotNull { authorNode ->
+                    when {
+                        authorNode.isObject -> authorNode["name"]?.textValue()
+                        authorNode.isTextual -> parseAuthorString(authorNode.textValue())
+                        else -> null
+                    }
+                }?.let { addAll(it) }
+            }
+
+        /**
+         * Parse the author if it is defined as a single string in the format "Name <mail> (url)". Try to extract the
+         * name part from the string or return the full string if no mail or url components are found.
+         */
+        private fun parseAuthorString(authorStr: String?): String? =
+            authorStr?.let { str ->
+                str.substringBefore('<').substringBefore('(').trim().ifBlank { null }
+            }
```

Comment:
Please wait until https://github.com/oss-review-toolkit/ort/pull/3684 is merged, then rebase, and make use to the new central `parseAuthorString()` function here.